docs/examples/node/aewallet.md

```diff
@@ -45,20 +31,33 @@ other addresses.
 
 'use strict'
 
-
-```
-
-
-
+const { Crypto, TxBuilder } = require('@aeternity/aepp-sdk')
+const program = require('commander')
+const fs = require('fs')
+const prompt = require('prompt')
+const path = require('path')
+
+function generateKeyPair (name, { output }) {
+  const { publicKey, secretKey } = Crypto.generateKeyPair()
+
+  const data = [
+    [path.join(output, name), secretKey],
+    [path.join(output, `${name}.pub`), publicKey]
+  ]
+
+  data.forEach(([path, data]) => {
+    fs.writeFileSync(path, data)
+    console.log(`Wrote ${path}`)
+  })
+}
```

Comment:
Have you copied it from [here](https://github.com/aeternity/aepp-sdk-js/blob/e8c55319c633dbf9317c7c6ed9dbeff57a1144ed/examples/node/aecrypto.js#L69-L81)? Actually, there is a separate example generated out of aecrypto.js, don't see reasons to duplicate it.